definitions.yml

```diff
@@ -275,3 +275,125 @@ definitions:
         type: string
       isTestIdp:
         type: boolean
+  BackendStatus:
+    type: object
+    description: "A complex object representing multiple things: backend status, section status (a.k.a IO-App banners), remote config"
+    required:
+      - is_alive
+      - message
+    properties:
+      config:
+        $ref: "#/definitions/Config"
```

Comment:
During the last implementation for the `enroll_bpd_after_add_payment_method`I was wondering why this section should be optional.
In my understanding, it contains numerous configurations that the app now uses, shouldn't it be mandatory?
This would also simplify the code that makes use of these types.

Reply:
Good point. I can't see any issues to change it and `sections` too
changed with https://github.com/pagopa/io-services-metadata/pull/403/commits/db45a1d3d319bb9968eea7e4be3992ec11c0708a